static/app/views/dashboardsV2/widgetCard/genericWidgetQueries.tsx

```diff
@@ -44,22 +51,18 @@ export type GenericWidgetQueriesProps<SeriesResponse, TableResponse> = {
   selection: PageFilters;
   widget: Widget;
   afterFetchSeriesData?: (result: SeriesResponse) => void;
-  afterFetchTableData?: (result: TableResponse) => void;
+  afterFetchTableData?: (
+    result: TableResponse,
+    response?: ResponseMeta
+  ) => void | {totalIssuesCount?: string};
```

Comment:
It is a little strange that we return `totalIssuesCount` here, can we combine this with any existing prop that stores totals or at least name it to be generic?